instrumentation/netty/netty-4.1/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/netty/v4_1/client/NettyHttpClientTracer.java

```diff
@@ -65,6 +66,21 @@ protected String responseHeader(HttpResponse httpResponse, String name) {
     return SETTER;
   }
 
+  public boolean shouldStartSpan(Context parentContext, HttpRequest request) {
+    if (!super.shouldStartSpan(parentContext)) {
+      return false;
+    }
+    // The AWS SDK uses Netty for asynchronous clients. We have not found how to propagate Context
+    // from the SDK instrumentation, which executes on an application thread, to Netty
+    // instrumentation, which executs on event loops. If it's possible, it may require instrumenting
```

Comment:
I think including a note here about the signature problem would be good too, since that's the biggest worry
```suggestion
    // instrumentation, which executes on event loops. If it's possible, it may require instrumenting
```